Delete Box.py

gyurcsovicsova opened the pull request.

gyurcsovicsova committed:
Delete Box.py

elratondesusi commented:
Nedá sa spustiť. Keď upravíš názov niečoho, musíš premenovať aj všetky jeho použitia.
Taktiež najprv si treba pozrieť najnovší pull request, kde sú urobené najnovšie zmeny v projekte. Skontrolovať ho, schváliť/neschváliť a v tomto kóde potom robiť svoje zmeny.

elratondesusi commented:
V BOX.py pre triedu box s nachádza atribút wrong a free, ak free znamená, že na boxe neparkuje momentálne žiadne auto, tak oba tieto atribúty tam potrebné nie sú. Ak totiž box ma zaznam aktualne None, tak je free=true a naopak. Wrong bolo dohodnuté nepoužívať a bude sa rátať podľa toho či sa firma parkujúca a prenajímajúca zhodujú.

gyurcsovicsova committed:
Add files via upload

gyurcsovicsova committed:
Add files via upload

elratondesusi commented:
https://github.com/TIS2019-FMFI/parking-system/pull/24/commits/22d73f2bd179c2fca998ab6d1c42ddb0129844b5
https://github.com/TIS2019-FMFI/parking-system/pull/24/commits/82860b51a63d99d18888db3c5f2bb68c31a4af78

Prosím riadiť sa všetkými komentármi...

> _Nedá sa spustiť. Keď upravíš názov niečoho, musíš premenovať aj všetky jeho použitia.
Taktiež najprv si treba pozrieť najnovší pull request, kde sú urobené najnovšie zmeny v projekte. Skontrolovať ho, schváliť/neschváliť a v tomto kóde potom robiť svoje zmeny._

Projekt sa stále nedá ani len spustiť...Keď už aj zmením niečo v kóde, aby sa spustiť dal vykazuje chyby, nedá sa otvoriť ani okno s info o boxe.
A rovnako aj s tým najnovším pull requestom, v projekte boli urobené medzitým zmeny, ktoré ty úplne ignoruješ.

gyurcsovicsova committed:
Add files via upload

fixnutý problém s otvaraním okna pre konkrétny box :) btw. ktoré zmeny myslíš, upravovala som aktuálnu verziu mastra, to mi príde ako "aktuálna" verzia (funkčná)

elratondesusi commented:
> fixnutý problém s otvaraním okna pre konkrétny box :)

okno sa otvorí, no nič v ňom nie je a zároveň vyhadzuje chybu.

>  btw. ktoré zmeny myslíš, upravovala som aktuálnu verziu mastra, to mi príde ako "aktuálna" verzia (funkčná)

Áno je to aktuálna verzia v mastrovi, no v pull requestoch čaká už 21 dní ďalšia na schválenie...Ako som už písala v predošlých komentároch, treba veci schvalovať alebo k nim písať komentáre s pripomienkami, nie vytvárať viacero verzií súborov.

gyurcsovicsova committed:
Add files via upload

marek98 reviewed (dismissed):
Okej

marek98 committed:
Update BOX.py

marek98 committed:
Merge branch 'master' into gyurcsovicsova-patch-1

marek98 committed:
Create Box.py~master

marek98 reviewed (approved):
okej
